docs/dppy/python-support-features.rst

```diff
@@ -0,0 +1,107 @@
+Supported Python features in DPPY kernel
+========================================
+
+This page lists the Python features supported in the DPPY Python. This includes all kernel and device functions compiled with ``@dppy.kernel``.
+
+Constructs
+==========
```

Comment:
```suggestion
==========
----------
```

Reply:
It makes better headers hierarchy. Please, make the same for the following headers too.